heareval/predictions/task_predictions.py

```diff
@@ -235,14 +273,35 @@ def task_predictions(
         )
     else:
         raise ValueError(f"embedding_type {metadata['embedding_type']} unknown.")
+    return predictor
+
+
+def task_predictions_test(
+    predictor: torch.nn.Module,
+    embedding_path: Path,
+    scene_embedding_size: int,
+    timestamp_embedding_size: int,
+):
+    metadata = json.load(embedding_path.joinpath("task_metadata.json").open())
+    label_vocab, nlabels = label_vocab_nlabels(embedding_path)
+
+    if metadata["embedding_type"] == "scene":
+        predictor = RandomProjectionPrediction(
```

Comment:
Don't we want to use the passed in predictor in this function?

Reply:
No the passed in predictor was used to create *embeddings*. We learn our own predictor on top. But we afterwards will change this to single hidden layer.

Reply:
I'm actually still a bit confused by this -- you pass in the predictor as a arg to this function and then overwrite it immediately? What's the point in passing in the predictor?